website/src/pages/en/token-api/quick-start.mdx

```diff
@@ -68,18 +68,19 @@ Make sure to replace `<token>` with the JWT Token generated from your API key.
 
 ## Chain and Feature Support
 
-| Network          |         evm-tokens          |        evm-uniswaps         |       evm-nft-tokens        |
-| ---------------- | :-------------------------: | :-------------------------: | :-------------------------: |
-| Ethereum Mainnet |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-| BSC              |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-| Base             |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-| Unichain         |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-| Arbitrum-One     | <Check alt="Supported" />\* | <Check alt="Supported" />\* | <Check alt="Supported" />\* |
-| Optimism         |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-| Polygon          |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-| Avalanche        |  <Check alt="Supported" />  |  <Check alt="Supported" />  |  <Check alt="Supported" />  |
-
-\*Some chains are still in the process of syncing. You may encounter `bad_database_response` errors or incorrect response values until data is fully synced.
+| Network          |          Tokens           |           Swaps           |           NFTs            |
+| ---------------- | :-----------------------: | :-----------------------: | :-----------------------: |
+| Ethereum Mainnet | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| BSC              | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Base             | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Unichain         | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Arbitrum-One     | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Optimism         | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Polygon          | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Avalanche        | <Check alt="Supported" /> | <Check alt="Supported" /> | <Check alt="Supported" /> |
+| Solana           | <Check alt="Supported" /> | <Check alt="Supported" /> |      ❌ Unsupported       |
+
+A full list of chain support for The Graph services can be found in [The Graph Docs](https://thegraph.com/docs/en/supported-networks/).
```

Comment:
```suggestion
A full list of chain support for The Graph services can be found in [The Graph Docs](/supported-networks/).
```

think it should be this way, right @benface ?

Reply:
Fixed in https://github.com/graphprotocol/docs/pull/998/commits/2c0a8efdac20c24f93f2263e9c8446dc4d494167